components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -22,91 +22,108 @@ To get the best possible IAM v2 experience, Chef Automate should be running the
-
-Applying deployment configuration
-  Started automate-gateway
-
 Creating default teams Editors and Viewers...
 
-Success: Enabled IAM v2
+Migrating existing teams...
+
+Success: Enabled IAM v2.1
 ```
 
-Note: To upgrade without porting existing policies, use the `--skip-policy-migration` flag: `chef-automate iam upgrade-to-v2 --skip-policy-migration`.
+To upgrade without porting existing policies, use: `chef-automate iam upgrade-to-v2 --skip-policy-migration`.
 
 ## View Policies
 
-After you've logged in to Chef Automate, select the **Settings** tab in the top navigation bar, then select and locate the `Policies` section on the left hand panel.
+After you've signed in to Chef Automate, select the **Settings** tab in the top navigation bar, then select and locate the `Policies` section in the left navigation.
 
 In this section, you can view all of your v2 policies. If you have upgraded without using the `--skip-policy-migration` flag, you will also see v1 policies.
 
 This includes the following:
 
-* New default (Chef-managed) policies: Administrator, Ingest, Editors, and Viewers.
+* New default, *Chef-managed* policies: Administrator, Ingest, Editors, and Viewers.
 * Imported v1 default policies--now called *legacy policies*--in the new v2 policy format and marked with the `[Legacy]` prefix.
 * Imported v1 custom policies that you created; these are marked with the `[Legacy]` prefix and a `(custom)` suffix.
 
 ![](/images/docs/admin-policies-migrated.png)
 
 ## Policy Conversion
 
-Now that you are up-and-running with v2 policies, as the first step we recommend that you reconstitute your v1 policies as v2 policies.
-Once that is done, then delete the old legacy v1 policies and you will have a clean, up-to-date system.
+After upgrading we recommend that you reconstitute your v1 policies as v2 policies.
+Once that is done, delete the old legacy v1 policies and you will have an up-to-date system.
+
+To delete a legacy policy, open the menu on any custom policy, located at the end of the policy row, and select **Delete Policy** from the menu. You will not have this option for Chef-managed policies.
+
+A warning appears if members are still attached to the policy because deleting that policy disrupts access for all of its members.
+However, you can still delete the policy.
+
+Alternately, if you include `--skip-policy-migration` on upgrade, no existing v1 policies will be migrated.
+You will still need to create new v2 policies to preserve any IAM behavior from v1.
 
-Alternately, you can run `--skip-policy-migration` on upgrade to start without any policies at all. You will still need to create new v2 policies to preserve any IAM behavior from v1.
+{{% warning %}}
+Note that several legacy policies, including *Compliance Profile Access* and *Ingest Access*,  have API tokens that will stop working if not ported.
+Alternately, you may choose to keep those policies intact if that's easier and you do not plan to alter those permissions.
```

Comment:
@susanev we talked about getting rid of this `Alternately` suggestion, right?

Reply:
oh right yes, will remove